WolvenKit/Views/Documents/RedDocumentView.xaml.cs

```diff
@@ -36,25 +36,7 @@ private void OnEditorDifficultyChanged(object sender, EditorDifficultyLevel leve
                 return;
             }
 
-            if (vm.IsDirty)
```

Comment:
why was this deleted?

Reply:
the reload isn't needed anymore to apply the layout changes (that was due to the broken bindings introduced by your PR that changed the SelectedItems from <object> to <CVM>).